External/SPMmodified/xASL/xASL_io_Nifti2Im.m

```diff
@@ -55,6 +55,11 @@
 
 niiMat = false; % default
 
+% Initialize the json output
+if nargout >= 2
```

Comment:
It's very minor of course, but I would from the perspective of prefering readability not do this clause.
There is no harm in always defaulting to json = [] right, irrespective of the nargout?
And code speed effect is negligible, it just makes for fewer if-statements which is desirable for bug-fixing in general.

So I would say that we in general default all variables to be empty when starting a function, and after the function checking for empty. And json = NaN means that there was a json but something went wrong in the loading (would also be nice to default that).

Reply:
Sure. no prob in always default to [].

I'd keep a simply output [] in case things go wrong. It's easier to check afterwards. And we of course print an error when this happens and this error is propagated, so the user notices that.